pallets/rmrk-core/Cargo.toml

```diff
@@ -51,6 +51,11 @@ version = '1.0'
 default-features = false
 path = '../uniques'
 
+[dependencies.rmrk-traits]
+default-features = false
+path = '../../traits'
+version = "0.4.1-dev"
```

Comment:
Is it necessary to specify the version given the dependency is a local path? Not sure if this will affect publishing the repo to crates.io. However in the early stage, most of the Substrate projects don't list on crates anyway...